templates/web/base/waste/index.html

```diff
@@ -1,9 +1,8 @@
 [% PROCESS 'govuk/fields.html' %]
 [% pre_container_extra = BLOCK %]
-
+    [% PROCESS errors %]
```

Comment:
It probably doesn't make much difference, but rather than have the div class="container" in the error block (and affecting everywhere else the error block is used, and I'm not sure how - looks like it'd be container inside container on a normal form page), could that be put here instead, and leave fields.html alone?

Reply:
(And as this is very unlikely to need to appear here, happy to keep the bottom margin the same for its display on other pages also)

Reply:
No, that's actually fine and more simple. That was my first approach, but I leaves an empty div in case there is no error(Which it's not terrible or a deal breaker). I tried replicating this:
```
[% IF error %]
    <p class="form-error">[% error %]</p>
[% END %]

// And leave it like:
 
[% IF error %]
    <div class="container">[% PROCESS errors  %]</div>
[% END %]

```
But it didn't work.